tests/unit-core/test-api-property.c

```diff
@@ -63,8 +63,8 @@ main (void)
 
   /* Test: get own property descriptor */
   prop_desc = jerry_property_descriptor_create ();
-  bool is_ok = jerry_get_own_property_descriptor (global_obj_val, prop_name, &prop_desc);
-  TEST_ASSERT (is_ok);
+  jerry_value_t is_ok = jerry_get_own_property_descriptor (global_obj_val, prop_name, &prop_desc);
```

Comment:
all `is_ok` need to be freed by `jerry_release_value`

Reply:
Do i need to free at the end or every time after i use it?

Reply:
Every time after no longer needed.